torch/_decomp/__init__.py

```diff
@@ -53,7 +53,18 @@ def add_op_to_table(aten_op):
                 if op_overload in registry:
                     raise RuntimeError(f"duplicate registrations for {op_overload}")
                 registry[op_overload] = f
-                if register_meta:
+                # TODO: factor this logic into OpOverload or Library API
+                name = op_overload._schema.name
+                if op_overload._schema.overload_name:
+                    name += "." + op_overload._schema.overload_name
+                if (
+                    not disable_meta
+                    # TorchScript dumps a bunch of extra nonsense overloads
+                    # which don't have corresponding dispatcher entries, we need
+                    # to filter those out
+                    and torch._C._dispatch_has_kernel(name)
```

Comment:
it would be nicer to add this filter to the `overloads()` method on `OpOverloadPacket since we wanted to disallow access to jit overloads anyway

Reply:
It's not obvious to me that this is the right thing to do. In particular, these overloads DO participate in torch.ops.aten.foo resolution, so it seems weird to filter them out from the API here.